utils/collectd/Makefile

```diff
@@ -9,7 +9,7 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=collectd
 PKG_VERSION:=5.12.0
-PKG_RELEASE:=$(AUTORELEASE)
+PKG_RELEASE:=2
```

Comment:
2 is wrong.
It should be 42.  (the always rigth answer ;-)

We are currently already at -41

collectd_5.12.0-41_aarch64_cortex-a53.ipk